docs/database/snippets/tutorial/aspiresqlefcore/AspireSQLEFCore.AppHost/Program.cs

```diff
@@ -0,0 +1,10 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var sqlpassword = builder.Configuration["sqlpassword"];
```

Comment:
Is this accurate?

Reply:
This is to solve the known bug described here: https://github.com/dotnet/aspire/issues/971 - the default setup doesn't work because the default SA password doesn't meet requirements, so we set a custom password in a user secret and retrieve it.